templates/kubernetes/docs/1.30/release-notes.md

```diff
@@ -0,0 +1,82 @@
+---
+wrapper_template: templates/docs/markdown.html
+markdown_includes:
+  nav: kubernetes/docs/shared/_side-navigation.md
+context:
+  title: 1.30 Release notes
+  description: Release notes for Charmed Kubernetes
+keywords: kubernetes, release, notes
+tags:
+  - news
+sidebar: k8smain-sidebar
+permalink: 1.30/release-notes.html
+layout:
```

Comment:
There is a bit of different format of list in repo here `layout: [base, ubuntu-com]`
Is there a difference in these two?